cmd/buildah/rm.go

```diff
@@ -31,27 +39,45 @@ func rmCmd(c *cli.Context) error {
 	}
 
 	var e error
-	for _, name := range args {
-		builder, err := openBuilder(store, name)
-		if e == nil {
-			e = err
-		}
+	if c.Bool("all") {
+		builders, err := openBuilders(store)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "error reading build container %q: %v\n", name, err)
-			continue
+			return errors.Wrapf(err, "error reading build containers")
 		}
 
-		id := builder.ContainerID
-		err = builder.Delete()
-		if e == nil {
-			e = err
+		for _, builder := range builders {
+			err = builder.Delete()
+			if e == nil {
+				e = err
+			}
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "error removing container %q: %v\n", builder.Container, err)
+				continue
+			}
 		}
-		if err != nil {
-			fmt.Fprintf(os.Stderr, "error removing container %q: %v\n", builder.Container, err)
-			continue
+	} else {
+		for _, name := range args {
+			builder, err := openBuilder(store, name)
+			if e == nil {
+				e = err
+			}
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "error reading build container %q: %v\n", name, err)
+				continue
+			}
+
+			id := builder.ContainerID
+			err = builder.Delete()
+			if e == nil {
+				e = err
+			}
+			if err != nil {
+				fmt.Fprintf(os.Stderr, "error removing container %q: %v\n", builder.Container, err)
+				continue
+			}
+			fmt.Printf("%s\n", id)
```

Comment:
diff wasn't very kind here.  The code in the else was all the pre-existing code and didn't change except for an extra leading tab.